UI/Components/PreviousSegment.cs

```diff
@@ -162,6 +162,7 @@ public void Update(IInvalidator invalidator, LiveSplitState state, float width,
             DeltaFormatter.Accuracy = Settings.DeltaAccuracy;
             DeltaFormatter.DropDecimals = Settings.DropDecimals;
             TimeSaveFormatter.Accuracy = Settings.TimeSaveAccuracy;
+            TimeSaveFormatter.DropDecimalsPossibleTimeSave = Settings.DropDecimalsPossibleTimeSave;
```

Comment:
Why can't you set `DropDecimals` instead?

Reply:
Oh, before I wasn't able to get this to work, but it is now with this change. I'll push in a bit and then https://github.com/LiveSplit/LiveSplit/pull/2212 can be closed.